styles/homemaincontent.css

```diff
@@ -0,0 +1,244 @@
+body {
+    min-height: 100%;
+    font-family: "Poppins", sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+  
+.container {
+    display: flex;
+    flex-grow: 1;
+}
+  
+.sidebar {
+    flex-basis: 25%;
+}
+  
+.main-content {
+    background-image: url("../images/pexels-c-reiners-907786.jpg") ;
+    background-repeat: no-repeat;
+    background-size: cover;
+    flex-grow: 1;
+    color: white;
+    animation: background-change 25s linear infinite;
+}
+  
+.content-title {
+    font-size: 2rem;
+    text-align: center;
+    text-transform: uppercase;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
You’re repeating the font family 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; multiple times. Consider declaring it once for body or a parent container (.main-content).

Reply:
will do

Reply:
done